var/spack/repos/builtin/packages/perfstubs/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Perfstubs(CMakePackage):
+    """Profiling API for adding tool instrumentation support to any project.
+
+This was motivated by the need to quickly add instrumentation to the
+[ADIOS2](https://github.com/ornladios/ADIOS2) library without adding a build
+dependency, or tying to a specific measurement tool.
+
+The initial prototype implementation was tied to TAU, but evolved to this more
+generic version, which was extracted as a separate repository for testing and
+demonstration purposes.
+"""
+
+    # proper url for your package's homepage here.
+    homepage = "https://github.com/khuck/perfstubs"
+    url      = "https://github.com/khuck/perfstubs.git"
```

Comment:
Drop this line if you aren't using a URL for downloading.

Reply:
If I omit it, spack gives a warning

Reply:
What's the warning message?